### PR TITLE
Rails fix

### PR DESCRIPTION
**Pull request by sacry-dyn**

### CODE REVIEW

**Comment by sacry-dyn** [comment]
@tobstarr please review if possible :)

File: push.go [hunk 1]
```diff
@@ -36,7 +36,17 @@ func (cmd *PushCommand) Run() error {
 			return err
 		}
 
+		formats, err := client.FormatsList(1, 50)
```
Comment (tobstarr) [inline push.go]: are we sure that 50 formats is enough?

File: push.go [hunk 2]
```diff
@@ -632,3 +643,37 @@ func (source *Source) GetLocaleID() string {
...
+		return *source.Params.FileFormat
+	}
+	if source.FileFormat != "" {
+		return source.FileFormat
+	}
+	return ""
+}
+
+func (source *Source) IncludesLocaleInformation() bool {
+	if source.Format != nil {
```
Comment (tobstarr) [inline push.go]: why not just do `return source.Format != nil && source.Format.IncludesLocaleInformation`

**Comment by tobstarr** [comment]
Is the API endpoint already deployed? We should not merge before that is the case!

**Comment by sacry-dyn** [comment]
@tobstarr API endpoint is open as PR, so not master yet

**Comment by sacry-dyn** [comment]
@tobstarr second review, we currently do not use the formats endpoint anymore, rather choosing more elegantly when to create a locale and when not. Also I accidentally renamed all RFC occurrences with Code (the naming is much better but not part of the PR)

**Pull request merged**